### DISCUSSION TITLE
verknüpfte Buchung im Mitgliedskonto mit Doppelklick öffnen

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#382] [General]
Lässt es sich irgendwie bewerkstelligen, dass man die im Mitgliedskonto angezeigten und verknüpften Buchungen per Doppelklick öffnen kann?
In den anderen Views geht das meistens. Ich möchte hin und wieder nachschauen, von welchem Konto aus überwiesen wurde. Dazu muss ich bisher immer in die Gesamtbuchungsliste wechseln.

**Comment by JohannMaierhofer** [comment]
Das wollte ich auch schon immer haben. Jetzt ist es da, siehe #383
